openquake/calculators/postproc/plots.py

```diff
@@ -59,6 +78,51 @@ def get_country_iso_codes(calc_id, assetcol):
     return id_0_str
 
 
+def plt_to_base64(plt):
+    bio = io.BytesIO()
```

Comment:
Why we are using base64? Is it required by the javascript side? Please add a comment.

Reply:
I added a docstring explaining it